From 4be4fe4bf6b117576a239a474a53ae495dfcc73d Mon Sep 17 00:00:00 2001
From: Balint Reczey <balint@balintreczey.hu>
Date: Thu, 7 Jul 2016 00:51:53 +0200
Subject: [PATCH 2/2] [StringUtils] Fix format string

---
 xbmc/utils/StringUtils.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/xbmc/utils/StringUtils.cpp
+++ b/xbmc/utils/StringUtils.cpp
@@ -1316,7 +1316,7 @@
 {
   const std::array<std::string, 6> units{"B", "kB", "MB", "GB", "TB", "PB"};
   if (bytes < 1000)
-    return Format("%lluB", bytes);
+    return Format("%"PRIu64"B", bytes);
 
   int i = 0;
   double value = static_cast<double>(bytes);
