From 5451bad08317e8f36d9541980f773ceeb7262e80 Mon Sep 17 00:00:00 2001
From: Balint Reczey <balint@balintreczey.hu>
Date: Fri, 7 Mar 2014 00:17:40 +0100
Subject: [PATCH] Disble static ffmpeg when using external ffmpeg/libav

---
 configure.in |    3 +++
 1 file changed, 3 insertions(+)

--- a/configure.in
+++ b/configure.in
@@ -876,6 +876,9 @@
     fi
   fi
 fi
+if test "$use_static_ffmpeg" = "yes" -a "$use_external_ffmpeg" = "yes"; then
+  use_static_ffmpeg=no
+fi
 if test "$use_static_ffmpeg" = "yes"; then
   USE_STATIC_FFMPEG=1
   AC_DEFINE([USE_STATIC_FFMPEG], [1], [link ffmpeg statically])
