From c52cc733710b343efbc2af808576da573b12b116 Mon Sep 17 00:00:00 2001
From: Balint Reczey <balint@balintreczey.hu>
Date: Fri, 7 Oct 2016 09:40:37 +0200
Subject: [PATCH 4/4] [utils] drop AsyncFileCopy from Makefile, too

---
 xbmc/utils/test/Makefile | 1 -
 1 file changed, 1 deletion(-)

diff --git a/xbmc/utils/test/Makefile b/xbmc/utils/test/Makefile
index a5e0a5f..496a8fb 100644
--- a/xbmc/utils/test/Makefile
+++ b/xbmc/utils/test/Makefile
@@ -2,7 +2,6 @@ SRCS=	\
 	TestAlarmClock.cpp \
 	TestAliasShortcutUtils.cpp \
 	TestArchive.cpp \
-	TestAsyncFileCopy.cpp \
 	TestBase64.cpp \
 	TestBitstreamStats.cpp \
 	TestCharsetConverter.cpp \
-- 
2.1.4

