From 4e15d0d88c014ad6bcf1243e9c28053aac4da5b9 Mon Sep 17 00:00:00 2001
From: Balint Reczey <balint@balintreczey.hu>
Date: Mon, 13 Oct 2014 17:10:40 +0100
Subject: [PATCH 5/6] mips: Don't use ASM rounding on MIPS

---
 xbmc/utils/MathUtils.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/xbmc/utils/MathUtils.h b/xbmc/utils/MathUtils.h
index ea32f37..82165f2 100644
--- a/xbmc/utils/MathUtils.h
+++ b/xbmc/utils/MathUtils.h
@@ -34,6 +34,7 @@
 
 #if defined(__ppc__) || \
     defined(__powerpc__) || \
+    defined(__mips__) || \
     defined(__arm__)
   #define DISABLE_MATHUTILS_ASM_ROUND_INT
 #endif
-- 
2.1.1

