From dc85016024fef825a8fffcd9881a7a0bd86912af Mon Sep 17 00:00:00 2001
From: Balint Reczey <balint@balintreczey.hu>
Date: Fri, 16 May 2014 12:07:51 +0200
Subject: [PATCH] Enable using external ffmpeg in ./configure

---
 configure.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/configure.in
+++ b/configure.in
@@ -602,7 +602,7 @@
 READELF="${READELF:-readelf}"
 NM="${NM:-nm}"
 
-use_external_ffmpeg=no
+use_external_ffmpeg=yes
 use_static_ffmpeg=no
 
 # ffmpeg needs the output of uname -s (e.x. linux, darwin) for the target_os
