--- a/addons/skin.confluence/720p/Home.xml
+++ b/addons/skin.confluence/720p/Home.xml
@@ -27,6 +27,18 @@
 			<include>Window_OpenClose_Animation</include>
 			<visible>!Skin.HasSetting(homepageWeatherinfo)</visible>
 		</control>
+		<control type="image">
+			<description>LOGO2</description>
+			<left>118</left>
+			<top>9</top>
+			<width>125</width>
+			<height>41</height>
+			<aspectratio aligny="top" align="left">keep</aspectratio>
+			<texture>from-debian-logo.png</texture>
+			<include>VisibleFadeEffect</include>
+			<include>Window_OpenClose_Animation</include>
+			<visible>!Skin.HasSetting(homepageWeatherinfo)</visible>
+		</control>
 		<!-- Music Info -->
 		<control type="group">
 			<left>0</left>
--- a/xbmc/GUIInfoManager.cpp
+++ b/xbmc/GUIInfoManager.cpp
@@ -4352,12 +4352,12 @@
   if (strlen(CCompileInfo::GetSuffix()) == 0)
     return StringUtils::Format("%d.%d", CCompileInfo::GetMajor(), CCompileInfo::GetMinor());
   else
-    return StringUtils::Format("%d.%d-%s", CCompileInfo::GetMajor(), CCompileInfo::GetMinor(), CCompileInfo::GetSuffix());
+    return StringUtils::Format("%d.%d-%s Debian package version:%s", CCompileInfo::GetMajor(), CCompileInfo::GetMinor(), CCompileInfo::GetSuffix(), DEB_VERSION);
 }
 
 std::string CGUIInfoManager::GetVersion()
 {
-  return GetVersionShort() + " Git:" + CCompileInfo::GetSCMID();
+  return GetVersionShort() + " Git: (unknown)";
 }
 
 std::string CGUIInfoManager::GetBuild()
@@ -4367,7 +4367,7 @@
 
 std::string CGUIInfoManager::GetAppName()
 {
-  return CCompileInfo::GetAppName();
+  return StringUtils::Format("%s from Debian", CCompileInfo::GetAppName());
 }
 
 void CGUIInfoManager::SetDisplayAfterSeek(unsigned int timeOut, int seekOffset)
