Description: Use system's libraries instead of the embedded ones
 .
Author: Balint Reczey <balint@balintreczey.hu>
Forwarded: not-needed

--- a/bootstrap.mk
+++ b/bootstrap.mk
@@ -1,6 +1,6 @@
 BOOTSTRAP_SUBDIRS += configure.ac
 BOOTSTRAP_SUBDIRS += lib/cpluff/configure.ac
-BOOTSTRAP_SUBDIRS += lib/gtest/configure.ac
+#BOOTSTRAP_SUBDIRS += lib/gtest/configure.ac
 
 ifneq ($(wildcard lib/libdvd/libdvdcss/configure.ac),)
 BOOTSTRAP_SUBDIRS += lib/libdvd/libdvdcss/configure.ac
--- a/configure.ac
+++ b/configure.ac
@@ -1053,7 +1053,7 @@
 AC_CHECK_HEADER([jpeglib.h],,        AC_MSG_ERROR($missing_library))
 AC_CHECK_HEADER([ogg/ogg.h],,        AC_MSG_ERROR($missing_library))
 AC_CHECK_HEADER([vorbis/vorbisfile.h],, AC_MSG_ERROR($missing_library))
-PKG_CHECK_MODULES([LIBDCADEC], [dcadec],, AC_MSG_ERROR([libdcadec not found]))
+#PKG_CHECK_MODULES([LIBDCADEC], [dcadec],, AC_MSG_ERROR([libdcadec not found]))
 
 PKG_CHECK_MODULES([LIBCURL], [libcurl],, AC_MSG_ERROR([libcurl not found]))
 XB_FIND_SONAME([CURL], [curl])
@@ -1190,6 +1190,9 @@
 PKG_CHECK_MODULES([TAGLIB],  [taglib >= 1.8],
   [INCLUDES="$INCLUDES $TAGLIB_CFLAGS"; LIBS="$LIBS $TAGLIB_LIBS"],
   AC_MSG_ERROR($missing_library))
+PKG_CHECK_MODULES([DVDREAD],  [dvdread],
+  [INCLUDES="$INCLUDES $DVDREAD_CFLAGS"; LIBS="$LIBS $DVDREAD_LIBS"],
+  AC_MSG_ERROR($missing_library))
 
 if test "$use_optical_drive" = "yes"; then
   PKG_CHECK_MODULES([CDIO],       [libcdio],
--- a/lib/cximage-6.0/Makefile.in
+++ b/lib/cximage-6.0/Makefile.in
@@ -4,8 +4,8 @@
 ARCHIVE=$(SO:.so=.a)
 SYSDIR=../../system
 SLIB=$(SYSDIR)/$(SO)
-DIRS=CxImage raw jbig
-SLIBS=CxImage/cximage.a raw/raw.a jbig/jbig.a
+DIRS=CxImage raw
+SLIBS=CxImage/cximage.a raw/raw.a
 
 ifeq ($(findstring osx,$(ARCH)), osx)
 	export ARCH
@@ -21,7 +21,7 @@
 		CxImage/*.o jasper/*/*.o jbig/*.o raw/*.o
 else
 	$(CXX) $(CFLAGS) $(LDFLAGS) -shared -o $(SLIB) -Wl,--whole-archive $(SLIBS) \
-		-Wl,--no-whole-archive -lpng -ljasper -ljpeg -ltiff -lz -Wl,--unresolved-symbols=ignore-all
+		-Wl,--no-whole-archive -ljbig -lpng -ljasper -ljpeg -ltiff -lz -Wl,--unresolved-symbols=ignore-all
 endif
 
 $(SLIBS): compile 
--- a/xbmc/cores/dvdplayer/DVDInputStreams/DllDvdNav.h
+++ b/xbmc/cores/dvdplayer/DVDInputStreams/DllDvdNav.h
@@ -34,6 +34,8 @@
  #define HAVE_CONFIG_H
  #endif
 
+ // pull in Kodi's config h instead of dvdnav's one
+ #include "xbmc/config.h"
  #include "dvdnav/dvdnav_internal.h"
  #include "dvdnav/vm.h"
  #include "dvdnav/dvd_types.h"
--- a/tools/depends/native/TexturePacker/Makefile
+++ b/tools/depends/native/TexturePacker/Makefile
@@ -9,13 +9,13 @@
 
 ifeq ($(NATIVEPLATFORM),)
   PLATFORM = native
-  EXTRA_CONFIGURE = --enable-static
+#  EXTRA_CONFIGURE = --enable-static
 else
   PLATFORM = $(NATIVEPLATFORM)
 endif
 
 ifeq ($(NATIVE_OS), linux)
-  EXTRA_CONFIGURE = --enable-static
+#  EXTRA_CONFIGURE = --enable-static
 endif
 ifeq ($(NATIVE_OS), android)
   EXTRA_CONFIGURE = --enable-static
