Description: Use system's libraries instead of the embedded ones
 .
Author: Balint Reczey <balint@balintreczey.hu>
Forwarded: not-needed

--- a/Makefile.in
+++ b/Makefile.in
@@ -187,7 +187,6 @@
 LIB_DIRS=\
 	lib/cximage-6.0 \
 	lib/libexif \
-	lib/libhdhomerun \
 	lib/cpluff \
 	lib/xbmc-dll-symbols
 
@@ -411,7 +410,7 @@
 
 codecs: dvdpcodecs dvdpextcodecs
 
-libs: $(LIBSSE4) libhdhomerun imagelib libexif system/libcpluff-@ARCH@.so
+libs: $(LIBSSE4) imagelib libexif system/libcpluff-@ARCH@.so
 
 externals: codecs libs visualizations screensavers libaddon
 
--- a/bootstrap.mk
+++ b/bootstrap.mk
@@ -2,7 +2,7 @@
 BOOTSTRAP_SUBDIRS += xbmc/screensavers/rsxs-0.9/configure.ac
 BOOTSTRAP_SUBDIRS += xbmc/visualizations/Goom/goom2k4-0/configure.ac
 BOOTSTRAP_SUBDIRS += lib/cpluff/configure.ac
-BOOTSTRAP_SUBDIRS += lib/gtest/configure.ac
+#BOOTSTRAP_SUBDIRS += lib/gtest/configure.ac
 
 ifneq ($(wildcard lib/libdvd/libdvdcss/configure.ac),)
 BOOTSTRAP_SUBDIRS += lib/libdvd/libdvdcss/configure.ac
--- a/configure.ac
+++ b/configure.ac
@@ -1218,6 +1218,9 @@
 PKG_CHECK_MODULES([TAGLIB],  [taglib >= 1.8],
   [INCLUDES="$INCLUDES $TAGLIB_CFLAGS"; LIBS="$LIBS $TAGLIB_LIBS"],
   AC_MSG_ERROR($missing_library))
+PKG_CHECK_MODULES([DVDREAD],  [dvdread],
+  [INCLUDES="$INCLUDES $DVDREAD_CFLAGS"; LIBS="$LIBS $DVDREAD_LIBS"],
+  AC_MSG_ERROR($missing_library))
 
 if test "$use_optical_drive" = "yes"; then
   PKG_CHECK_MODULES([CDIO],       [libcdio],
@@ -1312,6 +1315,7 @@
 XB_FIND_SONAME([VORBISFILE],  [vorbisfile])
 XB_FIND_SONAME([ASS],         [ass])
 XB_FIND_SONAME([MPEG2],       [mpeg2])
+XB_FIND_SONAME([HDHOMERUN],   [hdhomerun])
 
 # WebServer
 if test "$use_webserver" = "yes"; then
@@ -2477,7 +2481,6 @@
     xbmc/network/Makefile \
     xbmc/network/upnp/Makefile \
     lib/libexif/Makefile \
-    lib/libhdhomerun/Makefile \
     lib/cximage-6.0/Makefile \
     lib/libUPnP/Makefile \
     xbmc/DllPaths_generated.h \
--- a/lib/cximage-6.0/Makefile.in
+++ b/lib/cximage-6.0/Makefile.in
@@ -4,8 +4,8 @@
 ARCHIVE=$(SO:.so=.a)
 SYSDIR=../../system
 SLIB=$(SYSDIR)/$(SO)
-DIRS=CxImage raw jbig
-SLIBS=CxImage/cximage.a raw/raw.a jbig/jbig.a
+DIRS=CxImage raw
+SLIBS=CxImage/cximage.a raw/raw.a
 
 ifeq ($(findstring osx,$(ARCH)), osx)
 	export ARCH
@@ -21,7 +21,7 @@
 		CxImage/*.o jasper/*/*.o jbig/*.o raw/*.o
 else
 	$(CXX) $(CFLAGS) $(LDFLAGS) -shared -o $(SLIB) -Wl,--whole-archive $(SLIBS) \
-		-Wl,--no-whole-archive -lpng -ljasper -ljpeg -ltiff -lz -Wl,--unresolved-symbols=ignore-all
+		-Wl,--no-whole-archive -ljbig -lpng -ljasper -ljpeg -ltiff -lz -Wl,--unresolved-symbols=ignore-all
 endif
 
 $(SLIBS): compile 
--- a/xbmc/DllPaths_generated.h.in
+++ b/xbmc/DllPaths_generated.h.in
@@ -28,7 +28,7 @@
 #define DLL_PATH_CPLUFF        "special://xbmcbin/system/libcpluff-@ARCH@.so"
 #define DLL_PATH_IMAGELIB      "special://xbmcbin/system/ImageLib-@ARCH@.so"
 #define DLL_PATH_LIBEXIF       "special://xbmcbin/system/libexif-@ARCH@.so"
-#define DLL_PATH_LIBHDHOMERUN  "special://xbmcbin/system/hdhomerun-@ARCH@.so"
+#define DLL_PATH_LIBHDHOMERUN  "@HDHOMERUN_SONAME@"
 #define DLL_PATH_MEDIAINFO     "special://xbmcbin/system/mediainfo-@ARCH@.so"
 
 #define DLL_PATH_LIBRTMP       "@RTMP_SONAME@"
--- a/xbmc/filesystem/DllHDHomeRun.h
+++ b/xbmc/filesystem/DllHDHomeRun.h
@@ -20,7 +20,7 @@
  */
 
 #include "DynamicDll.h"
-#include "lib/libhdhomerun/hdhomerun.h"
+#include "libhdhomerun/hdhomerun.h"
 
 class DllHdHomeRunInterface
 {
--- a/xbmc/cores/dvdplayer/DVDInputStreams/DllDvdNav.h
+++ b/xbmc/cores/dvdplayer/DVDInputStreams/DllDvdNav.h
@@ -34,6 +34,8 @@
  #define HAVE_CONFIG_H
  #endif
 
+ // pull in Kodi's config h instead of dvdnav's one
+ #include "xbmc/config.h"
  #include "dvdnav/dvdnav_internal.h"
  #include "dvdnav/vm.h"
  #include "dvdnav/dvd_types.h"
