--- a/xbmc/utils/MathUtils.h
+++ b/xbmc/utils/MathUtils.h
@@ -35,6 +35,7 @@
 #if defined(__ppc__) || \
     defined(__powerpc__) || \
     defined(__mips__) || \
+   (defined(__VFP_FP__) && defined(__SOFTFP__)) ||       \
    (defined(TARGET_DARWIN_IOS) && defined(__llvm__)) || \
    (defined(TARGET_ANDROID) && defined(__arm__)) || \
     defined(TARGET_RASPBERRY_PI)
@@ -44,6 +45,7 @@
 #if defined(__ppc__) || \
     defined(__powerpc__) || \
     defined(__mips__) || \
+   (defined(__VFP_FP__) && defined(__SOFTFP__)) ||       \
    (defined(TARGET_DARWIN) && defined(__llvm__)) || \
    (defined(TARGET_ANDROID) && defined(__arm__)) || \
     defined(TARGET_RASPBERRY_PI)
