--- a/configure.ac
+++ b/configure.ac
@@ -2400,7 +2400,6 @@
     tools/Linux/${APP_NAME_LC}-standalone.sh \
     tools/Linux/${APP_NAME_LC}-xsession.desktop \
     tools/EventClients/Makefile \
-    tools/EventClients/Clients/OSXRemote/Makefile \
     xbmc/peripherals/bus/Makefile \
     xbmc/peripherals/devices/Makefile \
     xbmc/android/activity/Makefile \
--- a/xbmc/interfaces/builtins/ApplicationBuiltins.cpp
+++ b/xbmc/interfaces/builtins/ApplicationBuiltins.cpp
@@ -21,7 +21,9 @@
 #include "ApplicationBuiltins.h"
 
 #include "Application.h"
+#if defined(HAS_FILESYSTEM_RAR)
 #include "filesystem/RarManager.h"
+#endif
 #include "filesystem/ZipManager.h"
 #include "messaging/ApplicationMessenger.h"
 #include "interfaces/AnnouncementManager.h"
