diff --git a/configure.in b/configure.in
index 9337c1f..eb83e59 100644
--- a/configure.in
+++ b/configure.in
@@ -2509,7 +2509,6 @@ OUTPUT_FILES="Makefile \
     tools/Linux/FEH-ARM.py \
     tools/TexturePacker/Makefile \
     tools/EventClients/Makefile \
-    tools/EventClients/Clients/OSXRemote/Makefile \
     xbmc/peripherals/bus/Makefile \
     xbmc/peripherals/devices/Makefile \
     xbmc/android/activity/Makefile \
